README.md

```diff
@@ -23,4 +23,29 @@ This is a work in progress. If you find a mistake or have a suggestion, please [
 
 ## For 'step-04', 'step-05', 'step-06'
 
-Run `npm install` before running the code.
\ No newline at end of file
+Run `npm install` before running the code.
+
+## Frequently Asked Questions
+
+### Can I use WebRTC in PHP, Python or other backend languages?
+Yes. While this Codelab is built using Node.js in order to [enable signaling](https://www.html5rocks.com/en/tutorials/webrtc/infrastructure/) between peers, [WebRTC](https://webrtc.org) is a web API that can be used in any web server or backend.
+
+There are several repositories on GitHub built in [PHP](https://github.com/search?l=php&q=webrtc&type=Repositories) or [Python](https://github.com/search?l=Python&q=webrtc&type=Repositories) that can help you get started.
+
+### Can I use WebRTC in a server?
```

Comment:
I get what you're saying, but maybe 'Can I run a WebRTC app from a remote server?'

Reply:
Done.